app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ActiveRecord::Base
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post
+
+  def update_post_comments_counter
```

Comment:
- [ OPTIONAL ] I will suggest you apply my earlier comment here as well for the `update_post_comments_counter` method :wink: 
- Example
```ruby
`after_save :update_comments_counter`
```